chart/kubeapps/templates/dashboard/configmap.yaml

```diff
@@ -77,5 +77,6 @@ data:
       "featureFlags": {{ .Values.featureFlags | toJson }},
       "clusters": {{ template "kubeapps.clusterNames" . }},
       "theme": "{{ .Values.dashboard.defaultTheme }}",
-      "remoteComponentsUrl": "{{ .Values.dashboard.remoteComponentsUrl }}"
+      "remoteComponentsUrl": "{{ .Values.dashboard.remoteComponentsUrl }}",
+      "customAppViews": {{ .Values.dashboard.customAppViews | toJson }},
```

Comment:
There's a trailing comma that was added on the lastline above, which I assume is resulting in a json parsing error when the dashboard loads the config, but haven't verified.